worklist/views.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.shortcuts import render
+from worklist.models import WorkList
+
+
+# View to create a worklist
+def create_worklist(request):
+    content = {}
+    if request.method == 'POST':
+        content = {'error': 0, 'message': 'Successfully saved worklist'}
+        data = {'name': request.POST.get('name', ''),
```

Comment:
Should we really have a default value for name? Just disallow `''` entirely?

Reply:
I've edited it in the latest version